README.md

```diff
@@ -117,6 +124,8 @@ require('core-js/build')({
   // ...
 });
 ```
+If your webpack build includes something like UglifyJs, then the excluded parts will be entirely stripped from the resulting code using dead code elimination.
```

Comment:
The excluded parts will be missing *before* uglifyjs, so you can just remove this paragraph.